useragent/user_agent_test.go

```diff
@@ -74,3 +74,30 @@ func TestFromContext_Custom(t *testing.T) {
 func TestDefaultsAreValid(t *testing.T) {
 	WithProduct(productName, productVersion)
 }
+
+func TestSanitize(t *testing.T) {
```

Comment:
Nit: this belongs in `patterns_test.go`.

Reply:
done